src/lib/components/ui/withForwardedRef.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default (WrappedComponent) => {
+  const withForwardedRef = (props, ref) => (
+    <WrappedComponent {...props} forwardedRef={ref} />
+  );
+
+  withForwardedRef.displayName = `withForwardedRef(${WrappedComponent.name || WrappedComponent.name})`;
```

Comment:
@mbohal Reason is to display readable component name in DevTools and snapshots: https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools